tensorflow/lite/experimental/micro/examples/hello_world/arduino-esp32/output_handler.cc

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/experimental/micro/examples/hello_world/output_handler.h"
+
+#include "Arduino.h"
+#include "tensorflow/lite/experimental/micro/examples/hello_world/constants.h"
+
+// use first channel of 16 channels (started from zero)
+#define LEDC_CHANNEL_0     0
+
+// use 13 bit precission for LEDC timer
+#define LEDC_TIMER_13_BIT  13
+
+// use 5000 Hz as a LEDC base frequency
+#define LEDC_BASE_FREQ     5000
+
+// The pin of the built-in LED
+int led = LED_BUILTIN;
+
+// Track whether the function has run at least once
+bool initialized = false;
+
+// Set brightness of the (inbuilt) LED with PWM
+// This will give a slow fade-in and fade-out animation
+void HandleOutput(tflite::ErrorReporter* error_reporter, float x_value,
+                  float y_value) {
```

Comment:
This and the other suggestions will bring in line with the latest version of our hello_world Arduino sample, which has yet to be published.

```suggestion
                  float y_value) {
                  
                    // Track whether the function has run at least once
                    static bool is_initialized = false;
```